README.md

```diff
@@ -2,58 +2,57 @@
 
 Telegram bot for chats administration.
 
+[![Build Status](https://dev.azure.com/GrinderDotNetRu/Grinder/_apis/build/status/Keroosha.grinder?branchName=master)](https://dev.azure.com/GrinderDotNetRu/Grinder/_build/latest?definitionId=1&branchName=master)
+
 ## Configuration and build
 
-To build run you have to add a `bot_config.json` file on path `src\Grinder\`
+Just open project with Rider/VS 2019 and run build
 
-File structure:
+## Running
 
-```
-{
-  "Token": ...,
-  "ChannelId": ...,
-  "AdminUserId": ...,
-  "Socks5Proxy": {
-    "Hostname": "...",
-    "Port": ...,
-    "Username": "...",
-    "Password": "..."
-  },
-  "ChatsToMonitor": [
-    "chat @username",
-    ...
-  ],
-  "AllowedUsers": [
-    "username without at sign",
-    ...
-  ]
-}
-```
```

Comment:
And now, the documentation has no description of the JSON configuration file, but yet you offer one at `src/Grinder/appsettings.json`. What's the matter?

Reply:
@Liminiens Check dat please

Reply:
there is only bot_config.json and no appsettings.json